p2p/transport/websocket/websocket_test.go

```diff
@@ -548,3 +549,122 @@ func TestResolveMultiaddr(t *testing.T) {
 		})
 	}
 }
+
+func startListeners(tpt *WebsocketTransport) (ma.Multiaddr, []*manet.Listener, error) {
```

Comment:
Perhaps this function can be a `t.Helper()` and simplify some of the error handling?

Also, what is your opinion to using `fmt.Print*` family of functions in tests? I myself rarely use them, and use `t.Logf`. Those only get printed if the test fails or if the `verbose` output is enabled.

Reply:
Thanks, was not aware of this. Will make changes accordingly. 
I think t.Logf makes more sense in comparison to fmt.Print*. 
I started using fmt functions for debugging and then continued with them as is.